### PR TITLE
[ Marathi Translation ] Conflict Resolved from last commit

### PR DESCRIPTION
**Pull request by Prabhurajn**

### CODE REVIEW

File: config/translations/mr/config.delete.yml [hunk 1]
```diff
@@ -5,3 +5,4 @@ messages:
     enter-name: 'संरचनेच्या नावाचे मूल्य असणे आवश्यक आहे.'
     config-not-exists: 'संरचना "%s" अस्तित्वात नाही.'
     deleted: 'संरचना "%s" यशस्वीपणे हटविले.'
+
```
Comment (jmolivas) [inline config/translations/mr/config.delete.yml]: May you add a new empty line

File: config/translations/mr/config.import.single.yml [hunk 2]
```diff
@@ -1,11 +0,0 @@
-description: 'Import the selected configuration.'
```
Comment (jmolivas) [inline config/translations/mr/config.import.single.yml]: Why is this file removed from the repo?

Comment (Prabhurajn) [inline config/translations/mr/config.import.single.yml]: @jmolivas sorry this file was deleted by mistake

File: config/translations/mr/config.delete.yml (hunk 1 above)
Comment (Prabhurajn) [inline config/translations/mr/config.delete.yml]: i'll remove this line, this line is added by mistake by my IDE that m using

**Comment by Prabhurajn** [comment]
@jmolivas please check now

**Comment by jmolivas** [comment]
@Prabhurajn awesome, thanks for the PR.

PR was merged. This will be included on next release, Thanks.
